build/lin.sh

```diff
@@ -455,21 +455,22 @@ fi
 make install-strip
 
 mkdir ${DEPS}/cgif
-$CURL https://github.com/dloebl/cgif/archive/V${VERSION_CGIF}.tar.gz | tar xzC ${DEPS}/cgif --strip-components=1
+$CURL https://github.com/dloebl/cgif/archive/refs/tags/V${VERSION_CGIF}.tar.gz | tar xzC ${DEPS}/cgif --strip-components=1
 cd ${DEPS}/cgif
 CFLAGS="${CFLAGS} -O3" meson setup _build --default-library=static --buildtype=release --strip --prefix=${TARGET} ${MESON} \
   -Dtests=false
 ninja -C _build
 ninja -C _build install
 
 mkdir ${DEPS}/vips
-$CURL https://github.com/libvips/libvips/releases/download/v${VERSION_VIPS}/vips-${VERSION_VIPS}.tar.gz | tar xzC ${DEPS}/vips --strip-components=1
+$CURL https://github.com/libvips/libvips/archive/refs/tags/${VERSION_VIPS}.tar.gz | tar xzC ${DEPS}/vips --strip-components=1
```

Comment:
Is there something we can change/fix in the tarballs that libvips publishes to improve s390x support and avoid running autogen? (We're switching libvips to meson, so this problem might go away.)

Reply:
I don't think this is s390x specific. The tarball that is downloaded doesn't have configure built so you have to use autogen.sh to create it. The original URL wasn't finding the source tarball anymore.

Reply:
That problem was caused by me specifying v8.12.2 as an argument and not just 8.12.2. I'll push my stuff shortly. I just need to work out how I can integrate the s390x build/builder into the scheme.